pandas/core/config_init.py

```diff
@@ -314,12 +314,21 @@ def table_schema_cb(key):
     cf.register_option('column_space', 12, validator=is_int)
     cf.register_option('max_info_rows', 1690785, pc_max_info_rows_doc,
                        validator=is_instance_factory((int, type(None))))
-    cf.register_option('max_rows', 60, pc_max_rows_doc,
+    cf.register_option('max_rows', 20, pc_max_rows_doc,
                        validator=is_instance_factory([type(None), int]))
     cf.register_option('max_categories', 8, pc_max_categories_doc,
                        validator=is_int)
     cf.register_option('max_colwidth', 50, max_colwidth_doc, validator=is_int)
-    cf.register_option('max_columns', 20, pc_max_cols_doc,
+    try:
+        ip = get_ipython()
+    except NameError:  # assume standard Python interpreter in a terminal
```

Comment:
this logic should be in a function in ``pandas.io.formats.terminal`` (this can be imported at the top of the file as it doesn't have any dependencies)